e2e/cli-e2e/project.json

```diff
@@ -11,22 +11,23 @@
         "lintFilePatterns": ["e2e/cli-e2e/**/*.ts"]
       }
     },
-    "e2e-old": {
+    "nxv-e2e": {
+      "options": {
+        "environmentRoot": "static-environments/cli-e2e-env"
+      }
+    },
+    "nxv-env-setup": {
+      "options": {
+        "environmentRoot": "static-environments/cli-e2e-env"
+      }
+    },
+    "e2e": {
       "executor": "@nx/vite:test",
       "options": {
         "configFile": "e2e/cli-e2e/vite.config.e2e.ts"
       }
     }
   },
-  "implicitDependencies": [
-    "models",
-    "utils",
-    "core",
-    "cli",
-    "plugin-eslint",
-    "plugin-coverage",
-    "plugin-js-packages",
-    "react-todos-app"
-  ],
+  "implicitDependencies": ["cli", "cli-e2e-env"],
```

Comment:
Same feedback as for #871 and #872 - the environment root could just be sub-folder in `cli-e2e` for better colocation.